### ISSUE TITLE
Update to v1.08d-r5 breaks local key access

### ISSUE THREAD

**Issue by johnnydubrovnik** [PhilippC/keepass2android#1606]
Just saw that my Keepass2android version updated to v1.08d-r5. When I try to open my db using password + key file I get an error that it cannot access the local path where my key is. I checked permissions and the app has access to local media. I tried copying the key to another location and no joy.

I'm on Android 11, Pixel 3a XL. 

Help! All my work passwords are in there,

Thanks

**Comment by PhilippC** [comment]
what is the exact error message?

**Comment by johnnydubrovnik** [comment]
"Access to the path /storage/emulated/0/<rest of the path and key file> is denied"

The key file is there, and the app has permissions to access.

I copied the key online, changed the path and can access my passwords, but that’s a temporary workaround.

Thanks

**Comment by Gremlin69** [comment]
I had the same problem. The solution was easy in my case. For linking the local key file I used the EXTERNAL Browser which caused the error message " Access to the path /storage/emulated/0/(rest of the path and key file) is denied". When using the Keypass2Android browser the access to the key file worked. I used MIX Explorer, AMAZE, CX File Explorer. All were causing the same issue. Hope this helps.
